packages/fxa-payments-server/src/components/PaymentErrorView/index.stories.tsx

```diff
@@ -11,20 +12,25 @@ export default {
 
 const storyWithProps = () => {
   const story = () => (
-    <BrowserRouter>
-      <Routes>
-        <Route
-          path="*"
-          element={
-            <PaymentErrorView
-              error={{ code: 'general_paypal_error' }}
-              actionFn={() => {}}
-              plan={SELECTED_PLAN}
-            />
-          }
-        />
-      </Routes>
-    </BrowserRouter>
+    <AppLocalizationProvider
+      baseDir="./locales"
+      userLocales={navigator.languages}
+    >
```

Comment:
I'm using AppLocalizationProvider directly instead of withLocalization in some places because withLocalization causes a not very obvious type error with tsc when used in Payments and fxa-react.  So even though both `storybook` and `build-storybook` work fine (neither uses tsc), `build` fails with type errors.  I don't particularly want to go down that rabbit hole for this PR.